src/app/attendance/actions/attendanceActions.tsx

```diff
@@ -0,0 +1,26 @@
+"use server";
+import { AttendanceStatus, Table } from "@/types/index";
+import { getSupabaseClient } from "../../../../utils/supabase/supabaseClient";
+import { redirect } from "next/navigation";
+import { ATTENDANCE_PATH } from "@/constant/constant";
+
+type Attendance = {
```

Comment:
Where's this coming from? Shouldn't it be coming from Attendance component?

Reply:
Yes, fix